### PR TITLE
Added blog post for v0.12.2

### PR DESCRIPTION
**Pull request by karifabri**

### CODE REVIEW

File: content/posts/releases/v0.12.2-alpha.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+date: 2024-05-13
+title: "Announcing the release of Spice.ai v0.12.2-alpha"
```
Comment (lukekim) [inline content/posts/releases/v0.12.2-alpha.md:3]: Title in wrong format.

File: content/posts/releases/v0.12.2-alpha.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+type: blog
+linkTitle: "Spice.ai v0.12.2-alpha is now available"
+author: Spice AI ([@spice_ai](https://twitter.com/spice_ai))
+categories: [release]
+tags: [connectivity, snowflake, API, ODBC, append, SQL, connector, helm]
```
Comment (lukekim) [inline content/posts/releases/v0.12.2-alpha.md]:
```suggestion
tags: [error messages, snowflake, odbc, data refresh, helm]
```

File: content/posts/releases/v0.12.2-alpha.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+date: 2024-05-13
+title: "Announcing the release of Spice.ai v0.12.2-alpha"
+type: blog
+linkTitle: "Spice.ai v0.12.2-alpha is now available"
+author: Spice AI ([@spice_ai](https://twitter.com/spice_ai))
+categories: [release]
+tags: [error messages, snowflake, odbc, data refresh, helm]
+---
+
+# Spice v0.12.2-alpha (May 13, 2024)
```
Comment (lukekim) [inline content/posts/releases/v0.12.2-alpha.md]: Remove this, this should be in the `title` above.

File: content/posts/releases/v0.12.2-alpha.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
+date: 2024-05-13
+title: "Spice v0.12.1-alpha (May 6, 2024)"
```
Comment (sgrebnov) [inline content/posts/releases/v0.12.2-alpha.md]:
```suggestion
title: "Spice v0.12.2-alpha (May 13, 2024)"
```

**Review by lukekim — APPROVED** [review]

**Review by sgrebnov — APPROVED** [review]

**Pull request merged**